docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
     volumes:
       - ./services/:/services
     working_dir: /services/registry
-    command: sh -c "npm run prestart && npm run dev"
+    command: sh -c "npm run dev"
```

Comment:
So, I changed this to `npm run dev` because I assumed that `npm start` would be reserved for production and `nodemon` shouldn't run in production. 

The downside was that `prestart` no longer ran because `npm start` no longer used.[

[Commit](https://github.com/entropic-dev/entropic/commit/7492f789fb48a0a51ad3006782cc57b866492661#diff-4e5e90c6228fd48698d074241c2ba760R35)